### PR TITLE
Fix some spelling mistakes using codespell

### PR DESCRIPTION
**Pull request by alexfikl**
I was playing with [codespell](https://github.com/codespell-project/codespell) and wanted to use it on a larger project. These are the typos it found.

### CODE REVIEW

File: papis/base.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def basedoc_to_papisdoc(basedoc: Dict[str, Any]) -> Dict[str, Any]:
         _key_translate("dcsubject", "tags", "m", lambda x: " ".join(x)),
         _key_translate("dcdoi", "doi", "m", lambda x: x[0]),
         _key_translate("dctype", "type", "m", lambda x: x[0].lower()),
-        _key_translate("dclang", "lang", "mutli", lambda x: x[0]),
+        _key_translate("dclang", "lang", "m", lambda x: x[0]),
```
Comment (alexfikl) [inline papis/base.py:79]: Is this correct? Not sure what that `mutli` was and the code didn't seem to check for it..

(context: `codespell` drew my attention by wanting to spell it as `multi`)

Comment (alejandrogallo) [reply papis/base.py:79]: That is probably correct yes, thanks!

**Comment by alejandrogallo** [comment]
That's fine with me, tell me when it's ready

**Comment by alexfikl** [comment]
> That's fine with me, tell me when it's ready

Tests are passing nicely at
https://github.com/alexfikl/papis/actions/runs/1512977584
so this should be good to go from my end. Thanks for taking a look at all these PRs by the way!

**Comment by alejandrogallo** [comment]
thank you!

**Pull request merged**